pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -218,7 +218,7 @@ def test_offset_freqstr(self, offset_types):
 
         freqstr = offset.freqstr
         if freqstr not in ('<Easter>',
-                           "<DateOffset: kwds={'days': 1}>",
+                           "<DateOffset: days=1>",
```

Comment:
need a whatsnew note, explainig that the repr changed

Reply:
did the repr change break anything? can you add some tests for this (doesn't have to be comprehensive, but some basic ones is ok)